src/core_modules/capture-core/reducers/descriptions/searchPage.reducerDescription.js

```diff
@@ -1,6 +1,13 @@
 // @flow
-import { createReducerDescription } from '../../trackerRedux/trackerReducer';
-import { searchPageActionTypes } from '../../components/Pages/Search/SearchPage.actions';
+import { createReducer } from '@reduxjs/toolkit';
+import {
+    saveCurrentSearchInfo,
+    showEmptyResultsViewOnSearchPage,
+    showErrorViewOnSearchPage,
+    showInitialViewOnSearchPage,
+    showLoadingViewOnSearchPage,
+    showSuccessResultsViewOnSearchPage,
+} from '../../components/Pages/Search/SearchPage.actions';
```

Comment:
Again here we are only passing the action creator functions. We will use them to add cases to our reducers.